requirements.txt

```diff
@@ -1,6 +1,6 @@
 django-organizations>=1.0.0,<1.2.0
-django-extensions>=1.7.5,<2.2
-django-allauth>=0.35.0,<0.39.0
-django-phonenumber-field>=2.3.1,<2.4.0
-phonenumbers>=8.10.8,<8.11.0
+django-extensions>=1.7.5,<2.3
+django-allauth>=0.40
+django-phonenumber-field>=4.0
```

Comment:
always put an upper limit to the version to avoid future versions of the dependency to break the build